### ISSUE TITLE
[MAINT] Website fails to compile but action run does not fail

### ISSUE THREAD

**Issue by itellaetxe** [dipy/dipy.org#40]
As referenced in [PR #39](https://github.com/dipy/dipy.org/pull/39), sometimes there are cases when the webpage does not compile correctly, but the CI run shows no failure. See an example below (taken from [this run](https://github.com/dipy/dipy.org/actions/runs/9415155137/job/25935562148?pr=39#step:7:216)):

```
Warning, treated as error:
/home/runner/work/dipy.org/dipy.org/posts/2024/2024_06_07_Inigo_week_1.rst:17:Unexpected indentation.
```

_Originally posted by @skoudoro in https://github.com/dipy/dipy.org/issues/39#issuecomment-2154556697_ (adapted)

**Comment by skoudoro** [comment]
Thanks for creating the issue!

**Comment by skoudoro** [comment]
fixed by 7c7552d3c9ccf8ad826a768ff49d4c5bb0996a51 so closing
